process/data_processor.py

```diff
@@ -100,30 +101,38 @@ def fit_transform_subblocks(self, X_seq, term_weighting=None):
             # put into dataframe to add nas to missing events
             # divide by 20 as original operation multiplied by 20
             block_df = pd.DataFrame(block_counts, columns=self.events) / 20
+            block_df = block_df.reindex(sorted(block_df.columns), axis=1)
```

Comment:
sort the columns

I was getting random column order each time i ran it
which lead to the training set looking different from the test set